components/cam/src/dynamics/se/inidat.F90

```diff
@@ -475,6 +471,13 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
     end if
 
     if (fv_nphys == 0) then
+      ! Map phis data to FV physics grid
```

Comment:
This appears not to be right. The conditional at line 473 is `fv_nphys == 0`, but lines 474 to 479 then perform physgrid operations.

Reply:
oh, shoot, good catch. That's a pretty dumb mistake.